### ISSUE TITLE
VPN doesn't work on LP

### ISSUE THREAD

**Issue by gsstudios** [gsstudios/Dorimanx-SG2-I9100-Kernel#20]
VPN doesn't work on DorimanX kernel. However it works with stock LP kernels. 

Reports:
http://forum.xda-developers.com/showpost.php?p=65019183&postcount=713

**Comment by gsstudios** [comment]
@chhapil @realex-fire 

Any places where I should start looking? Any tips or info that would help me would be greatly appreciated. 

Thanks, josh

**Comment by gsstudios** [comment]
Will investigate this issue soon. Root cause has still been unidentified.

**Comment by Realex-fire** [comment]
@gsstudios
Try to change these lines in sbin/init: (initramfs3)

```
SYSTEM_FREE=$($BB df -h | $BB grep system | $BB cut -d " " -f14 | $BB cut -c 1-3);
# if [ "$SYSTEM_FREE" -le "492" ]; then
    $BB mv /lib/modules/*.ko /system/lib/modules/;
    $BB chmod 755 /system/lib/modules/*.ko;
    $BB ln -s /system/lib/modules/* /lib/modules/;
    $BB insmod /system/lib/modules/exfat.ko;
# else
#   $BB insmod /lib/modules/exfat.ko;
# fi;
```

Make so: (so was in kk branch)

```
SYSTEM_FREE=$($BB df -h | $BB grep system | $BB cut -d " " -f14 | $BB cut -c 1-3);
if [ "$SYSTEM_FREE" -le "492" ]; then
    $BB mv /lib/modules/*.ko /system/lib/modules/;
    $BB chmod 755 /system/lib/modules/*.ko;
    $BB ln -s /system/lib/modules/* /lib/modules/;
    $BB insmod /system/lib/modules/exfat.ko;
else
    $BB insmod /lib/modules/exfat.ko;
fi;
```

report about results and still try to launch cm11 on lp branch.
good luck ;)

**Comment by gsstudios** [comment]
Thanks, will test soon.

**Comment by gsstudios** [comment]
Ehm, it broke wifi. Probably best to keep it commented for now.

**Comment by gsstudios** [comment]
Should be solved in DorimanX 12.0 b002
